types/mysql/index.d.ts

```diff
@@ -535,37 +667,97 @@ export interface MysqlError extends Error {
 }
 
 export const enum Types {
```

Comment:
On this `Enum: Types`, I think we should leave it as is until we can precisely document them. What do you think ?

Reply:
that's fine with me